app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,17 @@
+# Dash-Board Controller
+class DashboardController < ApplicationController
+  def allnotes
+    @allnotes_hash = {}
+    @created_notes = Note.where(user_id: current_user.id)
+    @contributor_notes = Role.where(
+      user_id: current_user.id,
+      role: 'Contributor')
+    @reader_notes = Role.where(user_id: current_user.id, role: 'Reader')
+
+    @allnotes_hash = {
+      'Createdby' => @created_notes,
+      'Contributor' => @contributor_notes,
+      'Reader' => @reader_notes
+    }
```

Comment:
@sohilp393, Clean the code.

Reply:
@krishnasingh001. Code for dashboard_controller has been cleaned.
Also changed the name of the action from allnotes to index.